gitlab_pipeline_observability/.gitignore

```diff
@@ -0,0 +1,2 @@
+environment_*
+manifest_*
```

Comment:
we should also add `download_*` to ignore the configs downloaded if a merge of OpenPipeline configs would be necessary.